docker-compose.yml

```diff
@@ -13,29 +15,51 @@ services:
       - app
     volumes_from:
       - app
-    environment:
-      - VIRTUAL_HOST
+    env_file:
+      - frontend.env
     networks:
-    - back
-    - lb_web
+    - backend
+    - frontend-web
+    logging:
+      driver: none
   app:
     image: nextcloud:12-fpm
     links:
       - db
     volumes:
       - ./data/apps:/var/www/html/apps
       - ./data/config:/var/www/html/config
+      - ./data/custom_apps:/var/www/html/custom_apps
       - ./data/data:/var/www/html/data
     networks:
-    - back
+    - backend
+    logging:
+      driver: none
+  onlyoffice-document-server:
+    image: onlyoffice/documentserver
+    stdin_open: true
+    tty: true
+    env_file:
+      - onlyoffice.env
+    networks:
+      - frontend-web
+    expose:
+      - '80'
+    volumes:
+      - ./only/data:/var/www/onlyoffice/Data
+      - ./only/log:/var/log/onlyoffice
+    logging:
+      driver: none
   db:
-    image: mysql
+    image: mariadb
     volumes:
-      - ./mysql/runtime:/var/lib/mysql
+      - ./mariadb:/var/lib/mysql
     environment:
       - MYSQL_ROOT_PASSWORD
     networks:
-    - back
+    - backend
```

Comment:
I think it is fine to keep back and front, this is a little cosmetic thing.
If you want to change it, then we have to change it everywhere.